src/redux/categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'react-bookstore/categories/CHECK_STATUS';
+
+export default function categoryReducer(state = [], action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under construction';
+    default:
+      return state;
+  }
+}
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
```

Comment:
- [ ] I will suggest you to follow the project requirement according to the requirement Define a slice of state for categories that:
Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)